duxy/Day1/ej1.java

```diff
@@ -0,0 +1,36 @@
+import java.io.*;
+class ej1 {
+    public static void main(String[] args) {  
+        BufferedReader br = null;
+        try {
+            br = new BufferedReader(new FileReader("input.txt"));
+            String line;  
+            String all = "";
+            String[] parts = null;
```

Comment:
You don't need init this variable to null, simply use it in line [14](https://github.com/CloudCoders/AdventOfCode2017/pull/19/files#diff-d282bef7c3c00ea8a7f6a2b21b9cafc5R14)